### ISSUE TITLE
[Love is in the air] The Rocket's Pink Glare Achievement timer resets at 10s instead of 20s

### ISSUE THREAD

**Issue by UnePomme** [chromiecraft/chromiecraft#6672]
### What client do you play on?

enGB

### Faction

Alliance

### Content Phase:

Generic

### Current Behaviour

The Rocket's Pink Glare Achievement timer resets at 10 seconds instead of 20 seconds.
This nearly prevents you from getting the achievement unless using exactly off of GCD.

### Expected Blizzlike Behaviour

The achievement timer should be allowing 20 seconds to reach 10 shots until it resets.

### Source

![image](https://github.com/chromiecraft/chromiecraft/assets/44067875/5d9481d4-8472-4c51-85e3-d87abba00db1)

### Steps to reproduce the problem

1. Buy 2 packs of love rockets
2. Open the achievement window to keep track of progress
3. Shoot off the rockets
4. Count to 20

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/51c414b9d58c224c1a2238f8bfc07cbed0060a02

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by avarishd** [comment]
Works fine on my local, probably a module/CC specific issue?
![image](https://github.com/chromiecraft/chromiecraft/assets/46330494/f8bc4c30-7477-4875-9b6f-6a4e8a85cd04)

**Comment by UnePomme** [comment]
Just retested, and it looks like it's 12-13 seconds to reset actually. Still not 20 as it should be though.
Easier way to test, just shoot 1 rocket off and count until the achievement progress bar resets.

**Comment by avarishd** [comment]
Tested several times on my local (while having the achievement tracked and not), with the method you provided, all work as intended.
Additionally tested on CC PTR, still no issues.

**Comment by sudlud** [comment]
@UnePomme 
Could you please share a video showing the 12-13s reset?

**Comment by UnePomme** [comment]
I'll post a video tomorrow on Chromiecraft

**Comment by UnePomme** [comment]
I double checked to confirm, and either I was mistaken, or it has been fixed. I went to grab a video with a timer and it seems to be 20 second reset. I jumped on my main and was able to complete the achievement now. Leaning towards I was mistaken, but not sure why even when spamming them on my main, it would reset at 9 rockets shots every time (I tried 3 different times), however today I was just able to shoot off all 10 to receive the achievement. Closed as unable to reproduce
